scout/adapter/mongo/event.py

```diff
@@ -109,6 +109,7 @@ def create_event(
             event(dict): The inserted event
         """
         variant = variant or {}
+        current_time = datetime.now()
```

Comment:
Perhaps this line could be placed outside the functions, so it could be reused later, to avoid repetition?